akka-remote/src/main/protobuf/WireFormats.proto

```diff
@@ -64,7 +64,15 @@ message PropsData {
   required DeployData deploy = 2;
   required string clazz = 3;
   repeated bytes args = 4;
-  repeated string classes = 5;
+  // serialized props parameters
+  // older wire protocol: contains class name for each arg
+  // newer wire protocol: contains string manifest for each arg
+  repeated string manifests = 5;
+  // newer wire protocol: serializer id for each arg
+  repeated int32 serializerIds = 6;
```

Comment:
Ah, yes, true, I introduced a single repeating record. Manifests was already there though, it just changed name from classes, which is what the old one used, so what could be extracted to a record would be the serializer id and hasManifest flag, but then there'd be two arrays of single values and one with two values, better keep it consistent than having that mix IMO.